w2dynamics test failing

@ladinesa , @lauri-codes pointed out that the `test_w2dynamics.py` is failing after my last merge. The part failing is the last one about asserting the Fermi energy (it happens locally too):

```
 >       assert sec_scf[0].energy.fermi.to('eV').magnitude == approx(14.15922513307991)
E       AttributeError: 'NoneType' object has no attribute 'fermi'
```

Strangely, I was debugging with VSCode to this file, and I can retrieve the value as in the test. When uploading the file to the local nomad installation, I can see:

![Screenshot from 2023-07-20 13-08-42](https://github.com/nomad-coe/electronic-parsers/assets/112697669/65190347-f6e9-446b-a204-915bea24e43f)

I can deactivate the test, but if you have any ideas why, I can try to implement the fix.

--- a/tests/test_w2dynamicsparser.py
+++ b/tests/test_w2dynamicsparser.py
@@ -96,6 +96,3 @@ def test_srvo3(parser):
     # SCF tests
     sec_scf = sec_scc.scf_iteration
     assert len(sec_scf) == 4
-    assert sec_scf[0].energy.fermi.to('eV').magnitude == approx(14.15922513307991)
-    assert sec_scf[-1].energy.fermi.to('eV').magnitude == approx(14.1592279493078)
-    assert sec_scf[1].x_w2dynamics_ineq[0].x_w2dynamics_fiw == 'SrVO3_beta60_high-stat-2022-09-29-Thu-15-29-40.hdf5#dmft-002/ineq-001/fiw/value'

